src/api/status/gulpfile.js

```diff
@@ -0,0 +1,28 @@
+var gulp = require('gulp');
+var path = require('path');
+var sass = require('gulp-sass')(require('sass'));
+var autoprefixer = require('gulp-autoprefixer');
+var sourcemaps = require('gulp-sourcemaps');
+var open = require('gulp-open');
+
+var Paths = {
```

Comment:
Can we use `const` here?

Reply:
good catch, I just blindly copied that from the template.